reVX/plexos/plexos_cli.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-PLEXOS command line interface (cli).
+reV-ReEDS-PLEXOS command line interface (cli).
```

Comment:
maybe rename this file to rev_reeds_plexos_cli.py? Also I think cli should be a prefix when possible.

Reply:
renamed, though I like cli at the end...